### PR TITLE
a very simple example of writing a native appium test

### PR DESCRIPTION
**Pull request by kaochris**
I wrote a very simple example but I'm not sure if this is what's exactly needed. Please review and let me know if there needs to be changes.

### CODE REVIEW

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Hi @kaochris 

Can you turn this into a simple example instead of putting this in the BP folder?

Thanks!

**Comment by kaochris** [comment]
- Moved to examples folder
- Created files for running locally
- Created README for the example
- Updated the main README with a link to example

File: webdriverio/appium-app/examples/simple-example/test/configs/wdio.android.local.emu.conf.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import {join} from 'path';
+import config from './wdio.shared.local.appium.conf';
+
+config.capabilities = [
+    {
+        platformName: 'Android',
```
Comment (wswebcreation) [inline webdriverio/appium-app/examples/simple-example/test/configs/wdio.android.local.emu.conf.ts:6]: can we change this to W3C caps?

File: webdriverio/appium-app/examples/simple-example/test/configs/wdio.ios.local.sim.conf.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import {join} from  'path';
+import config from './wdio.shared.local.appium.conf';
+
+config.capabilities = [{
+    platformName: 'iOS',
```
Comment (wswebcreation) [inline webdriverio/appium-app/examples/simple-example/test/configs/wdio.ios.local.sim.conf.ts:5]: can we change this to W3C caps?

File: webdriverio/appium-app/examples/simple-example/test/configs/wdio.shared.conf.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+export const config: WebdriverIO.Config = {
```
Comment (wswebcreation) [inline webdriverio/appium-app/examples/simple-example/test/configs/wdio.shared.conf.ts:1]: can you also provide a link to where people can find the main config and the options?

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Hi @kaochris 

thanks, this looks good, I have some small remarks, can you take a look at them?

Thanks!

File: webdriverio/appium-app/examples/simple-example/test/configs/wdio.android.local.emu.conf.ts (hunk 1 above)
Comment (kaochris) [reply webdriverio/appium-app/examples/simple-example/test/configs/wdio.android.local.emu.conf.ts:6]: i thought platformName was W3C? Sorry, I'm new to W3C stuff.

**Review by wswebcreation — APPROVED** [review]
Looks good, thanks

**Pull request merged**